### PR TITLE
docs: Add timestamps to BLE protocol specification

### PR DESCRIPTION
**Pull request by Filiprogrammer**
Add time stamps to BLE characteristics which should arrive in the
correct order and might be re-sent by the client if the server does not
respond within a specified time frame.

### CODE REVIEW

**Review by LecraM2003 — DISMISSED** [review]

**Comment by Filiprogrammer** [comment]
#duration: 1

File: docs/wifi_ble_communication_protocol.md [hunk 1]
```diff
@@ -181,36 +181,42 @@ see [3.1. Toggle bit field](#31-toggle---bit-field)
 
 ### BLE_CHAR_TOGGLE (UUID: fce001d4-864a-48f4-9c95-de928f1da07b)
 
-see [3.1. Toggle bit field](#31-toggle---bit-field)
+| Field Name       | Field Type      | Notes                                                       |
+|------------------|-----------------|-------------------------------------------------------------|
+| Toggle bit field | uint8 array (2) | see [3.1. Toggle bit field](#31-toggle---bit-field)         |
```
Comment (LecraM2003) [inline docs/wifi_ble_communication_protocol.md:186]: Change uint8 array (2) to uint16

File: docs/wifi_ble_communication_protocol.md [hunk 2]
```diff
@@ -181,36 +181,43 @@ see [3.1. Toggle bit field](#31-toggle---bit-field)
 
 ### BLE_CHAR_TOGGLE (UUID: fce001d4-864a-48f4-9c95-de928f1da07b)
 
-see [3.1. Toggle bit field](#31-toggle---bit-field)
+| Field Name                | Field Type | Notes                                                                     |
+|---------------------------|------------|---------------------------------------------------------------------------|
+| Settings toggle bit field | uint8      | see [3.1.1. Toggle bit field (Settings)](#311-bit-field--1-byte-settings) |
+| Logging toggle bit field  | uint8      | see [3.1.1. Toggle bit field (Logging)](#312-bit-field--2-byte-logging)   |
```
Comment (LecraM2003) [inline docs/wifi_ble_communication_protocol.md:187]: correct 3.1.1. to 3.1.2.
```suggestion
| Logging toggle bit field  | uint8      | see [3.1.2. Toggle bit field (Logging)](#312-bit-field--2-byte-logging)   |
```

**Review by LecraM2003 — APPROVED** [review]

File: docs/wifi_ble_communication_protocol.md (hunk 1 above)
Comment (LecraM2003) [reply docs/wifi_ble_communication_protocol.md]: was changed in new commit

**Pull request merged**